scripts/build-mimalloc.sh

```diff
@@ -25,4 +25,4 @@ cmake "${CMAKE_FLAGS[@]}" . \
 
 ninja
 
-cp libmimalloc.a $BUN_DEPS_OUT_DIR/libmimalloc.a
+cp libmimalloc.a $BUN_DEPS_OUT_DIR/libmimalloc.o
```

Comment:
```suggestion
ar -x libmimalloc.a $BUN_DEPS_OUT_DIR/libmimalloc.o
```

Reply:
As I mentioned in my previous comment, this is not the intended behavior and adding this suggestion will break the build